packages/js/src/ui/Inbox.tsx

```diff
@@ -36,11 +42,12 @@ type InternalInboxProps = {
 
 const InternalInbox = (props: InternalInboxProps) => {
   const style = useStyle();
+  const { localized } = useLocalization();
 
   return (
     <div class={style('novu', 'root')}>
       <div class="nt-bg-primary-500 nt-p-3 nt-m-4">
-        <div class="nt-text-2xl nt-font-bold">Inbox</div>
+        <div class="nt-text-2xl nt-font-bold">{localized('inbox.title')}</div>
```

Comment:
Just a `t` 😆 It's kind of an industry standard!
```suggestion
        <div class="nt-text-2xl nt-font-bold">{t('inbox.title')}</div>
```